packages/eslint-plugin/tests/schema-snapshots/naming-convention.shot

```diff
@@ -0,0 +1,3473 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Rule schemas should be convertible to TS types for documentation purposes naming-convention 1`] = `
```

Comment:
we should refactor this one to use `$refs` as its duplicating a lot of stuff

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/deee1e97d58f34eda1ab46cdc41b2e73b3812a47/packages/eslint-plugin/tests/schema-snapshots/naming-convention.shot#L936-L1409